docs/sources/alert/_index.md

```diff
@@ -4,8 +4,6 @@ menuTitle: Alert
 description: Learn how the rule evaluates queries for alerting.
 aliases:
   - ./rules/
-  - ./alert/
-  - ./alerting/
```

Comment:
This could be a potentially valid alias. It would redirect /docs/loki/latest/alerting/ to this page.